documentation/Developers.md

```diff
@@ -19,6 +19,31 @@ Table of Contents
 A number of these can be installed using the
 [virtcontainers-setup.sh](../utils/virtcontainers-setup.sh) script.
 
+# Cloning and Submitting
+
+If you are just builiding `virtcontainers`, then you can simply clone the main repository
+from https://github.com/containers/virtcontainers.
+
+If you wish to develop `virtcontainers`, you should fork the project on github under your
+user.
+
+**Note:** As `virtcontainers` contains and references its own
+[sub packages](https://github.com/containers/virtcontainers/tree/master/pkg),
+it will **not** build or past its tests in your user fork repo (as the
+[pkg references](https://github.com/containers/virtcontainers/blob/master/cni.go#L25)
+will still reference the main repo, and this results in golang type clashes etc.).
+
+In order to develop and submit PRs for `virtcontainers`, the easist method is to:
+
+- create a user fork on github.
```

Comment:
I'd capitalise the first words in all these bullets.

Reply:
I agree.